Docs/README.md

```diff
@@ -64,65 +72,101 @@ Splice reads:                                  4161455
+This scripts guesses how the reads were stranded for strand specific RNA-seq data. 
+Example output from an unstranded(~50% sense/antisense) library of paired end data:
 
-```
-infer_experiment.py -r hg19.refseq.bed12 -i Pairend_nonStrandSpecific_36mer_Human_hg19.bam
+**From MultiQC report:**
+<img src= "images/infer_experiment.png">
 
-#Output:
+**From the infer_experiment.txt file:**
 
+```
 This is PairEnd Data
-Fraction of reads failed to determine: 0.0172
-Fraction of reads explained by "1++,1--,2+-,2-+": 0.4903
-Fraction of reads explained by "1+-,1-+,2++,2--": 0.4925
+Fraction of reads failed to determine: 0.0409
+Fraction of reads explained by "1++,1--,2+-,2-+": 0.4839
+Fraction of reads explained by "1+-,1-+,2++,2--": 0.4752
 ```
 
+
+
 ###junction_saturation.py
 This scripts aims to determine if the used sequencing depth is enough to perform  alternative splicing analysis on the data. 
 
-![Junction Saturation](images/junction_saturation.png)
+![Junction Saturation](images/Junction_saturation.png)
 
-In the above example output the red line (known junction) has reached a plateau. Nearly all known junctions in this data set have already been detected, and deeper sequencing will thus not help in detecting more known junctions. The green line (novel junction) has not plateaued and thus further sequencing will likely aid in junction saturation.
-In summary, if the lines are flattening out then you have sequenced deeply enough. 
-###RPKM saturation
+
+A line that has plateaued indicates that all junctions have been detected and that further secuencing will not yield more results. Insteads the same fragment would be sequenced again and again. None of the lines in the example above have plataued and thus these samples could reveal more alternative splicing information if they were sequenced deeper. 
+
+###RPKM_saturation.py
 Resamples a subset of the total RNA reads and calculates the RPKM value for each subset. We use the default subsets of every 5% of the total reads. I.e 5%,10%...95%,100%. 
 A percent relative error is then calcuated based on the subsamples, this is the y-axis in the graph. 
+As of this text being written there isn't a MultiQC submodule for displaying the output of RPKM_saturation.py. Below is instead the pdfs generated by the script itself:
 
 <img src= "images/saturation.png" width="500" height="500" >
 
 
-###read duplication
+###read_duplication.py
+This script investigates the read duplication rate. A good library will generally have a slightly declining graph. The red library in the example graph really stands out and needs investigating. 
+
+<img src= "images/rseqc_read_dups_plot.png" >
+
+###inner_distance.py
+inner_distance.py  tries to calculate the inner distance (or insert size) between two paired RNA reads. Calculated as _start of reads 2_ - _end of read 2_. 
+Note that values can be negative if the reads overlap, this is indicative of a poor library, and fragmented RNA. As in the example picture below. 
```

Comment:
_this is indicative of a poor library_ - not necessarily, it can be caused by a lot of different things.

> This script calculates the inner distance (or insert size) between two paired RNA reads. Note that this distance does _not_ include the sequenced read lengths, so numbers can be negative if the reads overlap.
>
> The library shown below has very short fragments, with most paired reads overlapping.